### ISSUE TITLE
Add ipv6 support for archive.turnkeylinux.org

### ISSUE THREAD

**Issue by QueerCodingGirl** [turnkeylinux/tracker#257]
Our servers run on ipv6 only and it would be nice to use turnkey... but there is no AAAA dns record for archive.turnkeylinux.org.

please add ipv6 to your hosts, thanks.
##

**Comment by JedMeister** [comment]
WTF?!? This still hasn't been done! It's 2018 and we still don't have IPv6 access to our package repo!

This needs fixing urgently!

**Comment by DarkGigaByte** [comment]
Yes it does... since 2014!

**Comment by JedMeister** [comment]
FWIW, I queried @lirazsiri on this. He reported that he did indeed enable it at some point (circa late 2014; not that long after his last post apparently). But there were some unintended consequence (not sure what they were?) so it got disabled again...

I have asked for further clarification and have noted, that this will need to be resolved at some point. So if not now, when? We'll see...

**Comment by OnGle** [comment]
Ping @JedMeister could I get a status update on this?

**Comment by JedMeister** [comment]
Apologies on the radio silence on this. Liraz has moved off to another project (he's still involved in TurnKey but not so actively). I'll follow up internally and at least get some clarification on why this hasn't moved. Frankly in late 2018 it's a bit embarrassing that IPv6 isn't available for our repo...

**Comment by Duckbuster** [comment]
Was experimenting with proxmox and turnkey lxc's.
Was gonna do ipv6 only lxc's so I ran into this issue as well.

Please do a check on why this hasn't happened already, and what the issue was exactly.

Thank you in advance!

**Comment by JedMeister** [comment]
FWIW, I've just sent out another message asking why this hasn't happened yet and asking if there is someway that I can assist push this forward...

**Comment by JedMeister** [comment]
Ok, so I now understand the issue a bit better. I think I have a workaround, but unfortunately, I don't yet have the spare cycles to test and/or implement it. The (upcoming and well overdue) v16.0 release is my current priority ([currently in RC](https://www.turnkeylinux.org/blog/v16.0rc1-core-and-tkldev) but stable should be released really soon). Hopefully once the dust settles on that, I will be able to circle back to this and do some testing...

**Comment by OnGle** [comment]
@JedMeister  what's the status on this?

I re-added critical label because this, if still an issue is actively making it impossible for users to use turnkeylinux and has done so for the last 7 years. I'm grudgingly moving this to 17.0 as 16.1 has long past but I heavily suggest we put the effort into fixing this as a top priority.

**Comment by QueerCodingGirl** [comment]
Would be great... i am still waiting on this one^^
And right now a mayor german hoster (Hetzner) has announced a massive price change for ipv4 addresses. There will be an increased need for german turnkey users to get this fixed.

**Comment by OnGle** [comment]
@alonswartz I know you've got a lot on your plate atm, but any chance we could get this fixed before 17 stable? This issue has been here for 8 years and afaik you're the only one who can handle this.

**Comment by codedbreeze** [comment]
Hi, any updates on IPv6 support? Or is it entirely impossible to implement?

**Comment by JedMeister** [comment]
> Hi, any updates on IPv6 support? Or is it entirely impossible to implement?

Unfortunately it's not currently possible with our infrastructure set up. We do have a workaround designed, but we just haven't had the spare cycles to implement it.

I hope to circle back to this once we (finally) complete the v17.x release.

**Comment by codedbreeze** [comment]
Thanks for the update @JedMeister. Please be sure to provide an update when the workaround is finally implemented.

**Comment by JedMeister** [comment]
This is still unresolved... It got to the top of my todo list, then I had to reprioritise to focus on v18.0. Once v18.0 is released, I hope to circle back to this one...
